Validation_Tests/Demonstration/Basic/BasicSongValidator.cs

```diff
@@ -1,28 +1,100 @@
-﻿using PopValidations;
-
-namespace PopValidations_Tests.Demonstration.Basic;
-
-public class BasicSongValidator : AbstractValidator<BasicSong>
-{
-    public BasicSongValidator()
-    {
-        Describe(x => x.TrackNumber)
-            .NotNull();
-
-        Describe(x => x.TrackName)
-            .IsEqualTo("Definitely Not The Correct Song Name.");
-
-        Describe(x => x.Duration)
-            .IsEqualTo(
-                -1, 
-                o => o
-                    .WithErrorMessage("Song must have a negative duration.")
-                    .WithDescription("Songs must force you to travel slowly backwards in time to listen to.")
-            );
-
-        Describe(x => x.Genre)
-            //.Vitally().NotEmpty();
-            //.HasLengthBetween(20, 400)
-            ;
-    }
+﻿using PopValidations;

```

Comment:
Nice.
Im thinking later, might be worth doing a demonstration file for each validator, so its easier to keep the code in sync.
But it will have to be a test that works against the swagger thingo.
So slightly more complicated.